test/test_device_thermal_generation_constructors.jl

```diff
@@ -572,7 +572,6 @@ end
     )
     @test build!(ED; output_dir = mktempdir(cleanup = true)) == PSI.BuildStatus.BUILT
     moi_tests(ED, false, 10, 0, 20, 10, 5, false)
-    res = solve!(ED)
```

Comment:
These are removed to avoid errors with double calls to `solve`.  The function `psi_checksolve_test` also calls solve.